src/coreclr/vm/exstate.h

```diff
@@ -264,48 +264,4 @@ class ThreadExceptionFlagHolder
-// the object would have the IP inside it.
-#define CAPTURE_BUCKETS_AT_TRANSITION(pThread, oThrowable)                                                                                  \
-        if (IsWatsonEnabled())                                                                                                              \
-        {                                                                                                                                   \
-            /* Switch to COOP mode */                                                                                                       \
-            GCX_COOP();                                                                                                                     \
-                                                                                                                                            \
-            /* oThrowable is actually GET_THROWABLE macro; so extract the actual throwable for once and for all */                          \
-            OBJECTREF throwable = oThrowable;                                                                                               \
-            if (CLRException::IsPreallocatedExceptionObject(throwable))                                                                     \
-            {                                                                                                                               \
-                if (pThread->GetExceptionState()->GetCurrentExceptionTracker() != NULL)                                                     \
-                {                                                                                                                           \
-                    if (!IsThrowableThreadAbortException(throwable))                                                                        \
-                    {                                                                                                                       \
-                        PTR_EHWatsonBucketTracker pWatsonBucketTracker = GetWatsonBucketTrackerForPreallocatedException(throwable, FALSE);  \
-                        if (pWatsonBucketTracker != NULL)                                                                                   \
-                        {                                                                                                                   \
-                            pThread->GetExceptionState()->GetUEWatsonBucketTracker()->CopyEHWatsonBucketTracker(*(pWatsonBucketTracker));   \
-                            pThread->GetExceptionState()->GetUEWatsonBucketTracker()->CaptureUnhandledInfoForWatson(TypeOfReportedError::UnhandledException, pThread, NULL); \
-                            DEBUG_STMT(pThread->GetExceptionState()->GetUEWatsonBucketTracker()->SetCapturedAtADTransition();)              \
```

Comment:
Is `SetCapturedAtADTransition` method and related flags unused as well now?

Reply:
It is still being used in AppDomainTransitionExceptionFilter, which is used in DispatchCallDebuggerWrapper. But now I've found that DispatchCallDebuggerWrapper is not used anymore, so I'll add more removals to this PR.

Reply:
Hmm, I've overlooked that the DispatchCallDebuggerWrapper is actually still used.